packages/at_chops/lib/src/algorithm/default_encryption_algo.dart

```diff
@@ -1,32 +1,45 @@
 import 'dart:typed_data';
 
 import 'package:at_chops/src/algorithm/at_algorithm.dart';
-import 'package:at_chops/src/algorithm/at_iv.dart';
+import 'package:at_chops/src/key/at_private_key.dart';
+import 'package:at_chops/src/key/at_public_key.dart';
 import 'package:at_chops/src/key/impl/at_encryption_key_pair.dart';
 import 'package:at_commons/at_commons.dart';
 import 'package:crypton/crypton.dart';
 
-class DefaultEncryptionAlgo implements AtEncryptionAlgorithm {
-  final AtEncryptionKeyPair _encryptionKeypair;
-  DefaultEncryptionAlgo(this._encryptionKeypair);
-
+class DefaultEncryptionAlgo implements ASymmetricEncryptionAlgorithm {
```

Comment:
Should this class now be called RsaEncryptionAlgo?

Reply:
yes. done the changes